admin_app/src/components/DownloadModal.tsx

```diff
@@ -15,6 +15,19 @@ import { LoadingButton } from "@mui/lab";
 
 const MAX_CARDS_TO_FETCH = 200;
 
+interface Content {
```

Comment:
Also defined in "@/app/content/edit/page".
Can we just define it in one place and import it where needed?

Reply:
Will avoid bugs where we change one and forget to change the other